gems/aws-sdk-core/lib/aws-sdk-core/plugins/sign.rb

```diff
@@ -13,8 +13,7 @@ class Sign < Seahorse::Client::Plugin
       option(:sigv4_region)
       option(:unsigned_operations, default: [])
 
-      supported_auth_types = %w[sigv4 bearer sigv4-s3express sigv4a none]
-      SUPPORTED_AUTH_TYPES = supported_auth_types.freeze
```

Comment:
Prior to this, I believe we added sigv4a if CRT was loaded, but now we just support everything.